File: backend/testing/corpus/javascript/archivo2.js
// Manejo de datos estáticos sin interacción externa

const usuarios = [
    { id: 1, nombre: "Ana" },
    { id: 2, nombre: "Luis" },
    { id: 3, nombre: "María" }
];

function listarUsuarios() {
    usuarios.forEach(function(usuario) {
        console.log("ID:", usuario.id, "| Nombre:", usuario.nombre);
    });
}

function contarUsuarios() {
    return usuarios.length;
}

listarUsuarios();
console.log("Total de usuarios:", contarUsuarios());

File: backend/testing/corpus/python/archivo2.py


# Manejo de datos internos sin interacción externa

libros = [
    {"id": 1, "titulo": "Python Básico"},
    {"id": 2, "titulo": "Seguridad Informática"},
    {"id": 3, "titulo": "Redes"}
]

def listar_libros():
    for libro in libros:
        print(f"ID: {libro['id']} | Título: {libro['titulo']}")

def contar_libros() -> int:
    return len(libros)

if __name__ == "__main__":
    listar_libros()
    print("Total de libros:", contar_libros())






